contractcourt/htlc_timeout_resolver_test.go

```diff
@@ -870,16 +876,12 @@ func TestHtlcTimeoutSecondStageRemoteSpend(t *testing.T) {
 	}
 
 	checkpoints := []checkpoint{
-		{
-			// Output should be handed off to the nursery.
-			incubating: true,
-		},
 		{
 			// We send a confirmation for the remote's second layer
 			// success transcation.
 			preCheckpoint: func(ctx *htlcResolverTestContext,
-				_ bool) error {
-
+				_ bool,
+			) error {
```

Comment:
Hmm, I'm using a new editor set up, looks like it has some aggressive line folding.

Reply:
Yeah, was about to comment on that as well... Still very much struggling with not hating this way of formatting function/method definitions. It just feels wrong somehow (probably just because I'm so used to our current style).

Can this perhaps be configured in your IDE? The way things are wrapped as well as how aggressively?
Also, since this has crept into the code base already, I still think we should update our `docs/code_formatting_rules.md` to specify when this formatting is allowed vs. the current rules.

Reply:
So it was using this: https://github.com/mvdan/gofumpt

I've turned it off for now, so it's just OG `gofmt`+`goimports`. Will manually roll back the formatting changes in the PR.